cli/azd/pkg/apphost/testdata/TestAspireContainerGeneration-resources.bicep.snap

```diff
@@ -5,18 +5,6 @@ param location string = resourceGroup().location
 param tags object = {}
 
 var resourceToken = uniqueString(resourceGroup().id)
-@secure()
```

Comment:
These are still present on `main.bicep` and thus we will prompt for / generate them, and can refer to them later via our templates.  But they are no longer needed on `resources.bicep` since we no longer create container apps here.